libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -185,6 +186,30 @@ AP_BattMonitor::init()
             case Type::Generator:
                 drivers[instance] = new AP_BattMonitor_Generator(*this, state[instance], _params[instance]);
                 break;
+            case Type::INA3221_CH1:
+//                _params[instance]._i2c_bus.set_default(AP_BATTMONITOR_SMBUS_BUS_INTERNAL),
+                drivers[instance] = new AP_BattMonitor_INA3221(
+                    *this,
+                    state[instance],
+                    _params[instance],
+                    Type::INA3221_CH1);
```

Comment:
As noted before the type really doesn't look to be needed here, and then the arguments can all fit on one line the same as the others around it.

Reply:
fixed